locales/de/common.json

```diff
@@ -0,0 +1,34 @@
+{
+  "title": "F1 Kalender",
+  "subtitle": "Rennen, Qualifikation & Trainings",
+  "meta":{
+    "description": "Formel 1 Kalender für die Saison {{year}} mit allen Großen Preisen der Formel 1, Trainings &amp; Qualifikationen. Funktion für Erinnerungen. Alle Zeitzonen. Herunterladen oder abonnieren.",
+    "keywords": "F1, Formel 1, Rennzeiten, Rennen, Erinnerungen, Alarme, Große Preise, Großer Preis, Kalender, Daten, Startzeiten, Qualifikation, Training, London, Europa, {{year}}"
+  },
+  "options": {
+    "calendar": "Füge diese Renndaten - und Zeiten zu deinem Handy, Kalender, Outlook oder Google Calendar hinzu",
```

Comment:
why the dash in the middle, would leave if out

Reply:
also would translate google calendar to google kalender

Reply:
> why the dash in the middle, would leave if out

Because it is 'Rennzeiten' and 'Renndaten'.